web/js/containers/animation-widget.js

```diff
@@ -292,6 +317,13 @@ class AnimationWidget extends React.Component {
       + `${hasSubdailyLayers ? 'subdaily ' : ''}`
       + `${isMobile ? 'mobile ' : ''}`
       + `${isLandscape ? 'landscape ' : ''}`;
+    const subdailyID = hasSubdailyLayers ? '-subdaily' : '';
+
+    const widgetIDs = (isMobilePhone && isPortrait) || (screenWidth < 670 && hasSubdailyLayers) || (screenWidth < 575 && !hasSubdailyLayers) ? `collapsed-animate-widget-phone-portrait${subdailyID}`
```

Comment:
This is another very verbose nested definition that could be broken down and simplified.  Are any of these checks the same across components?  Can a helper function be created somewhere else to avoid redefining these conditions?